src/code/z_scene.c

```diff
@@ -422,13 +399,11 @@ void func_80099140(GlobalContext* globalCtx, SceneCmd* cmd) {
     }
 }
 
-// Scene Command 0x16: Echo Setting
-void func_8009918C(GlobalContext* globalCtx, SceneCmd* cmd) {
+void Scene_CommandEchoSettings(GlobalContext* globalCtx, SceneCmd* cmd) {
     globalCtx->roomCtx.curRoom.echo = cmd->echoSettings.echo;
 }
```

Comment:
Same here, both `Echo`/`Reverb` is used interchangeably, and in other places as well. So just making a note that they should be made consistent